modules/installation-requirements-ibm-z.adoc

```diff
@@ -62,22 +62,21 @@ The following {ibm-name} hardware is supported with {product-title} version {pro
 2. The {op-system-base} KVM host in your environment must meet certain requirements to host the virtual machines that you plan for the {product-title} environment. See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_and_managing_virtualization/getting-started-with-virtualization-in-rhel-8_configuring-and-managing-virtualization[Getting started with virtualization].
 
 [discrete]
+[id="ibm-z-hardware-requirements_{context}"]
 == Hardware requirements
 
 * The equivalent of six Integrated Facilities for Linux (IFL), which are SMT2 enabled, for each cluster.
 * At least one network connection to both connect to the `LoadBalancer` service and to serve data for traffic outside the cluster.
 
-[NOTE]
-====
-You can use dedicated or shared IFLs to assign sufficient compute resources. Resource sharing is one of the key strengths of {ibm-z-name}. However, you must adjust capacity correctly on each hypervisor layer and ensure sufficient resources for every {product-title} cluster.
-====
-
 [IMPORTANT]
 ====
-Since the overall performance of the cluster can be impacted, the LPARs that are used to set up the {product-title} clusters must provide sufficient compute capacity. In this context, LPAR weight management, entitlements, and CPU shares on the hypervisor level play an important role.
+* You can use dedicated or shared IFLs to assign sufficient compute resources. Resource sharing is one of the key strengths of {ibm-z-name}. However, you must adjust capacity correctly on each hypervisor layer and ensure sufficient resources for every {product-title} cluster.
+
+* Since the overall performance of the cluster can be impacted, the LPARs that are used to set up the {product-title} clusters must provide sufficient compute capacity. In this context, LPAR weight management, entitlements, and CPU shares on the hypervisor level play an important role.
```

Comment:
"play an important role" is not informative; do we have specific guidance for these parameters?

Reply:
yes we have an extensive doc here: https://docs.openshift.com/container-platform/4.16/scalability_and_performance/ibm-z-recommended-host-practices.html

Reply:
In that case, could this be made more informative (to Shauna's point) by saying something like "For more information, see Recommended host practices for IBM Z & IBM LinuxONE environments" and including an additional resources link?